src/Command/PullRequest/PullRequestMergeCommand.php

```diff
@@ -388,4 +375,40 @@ private function addClosedPullRequestNote(array $pr, $mergeCommit, $squash = fal
 
         $this->getAdapter()->createComment($pr['number'], $this->render($template, $params));
     }
+
+    private function givePatToPullRequestAuthor(array $pr, $pat)
+    {
+        $config = $this->getConfig();
+        $configuredPat = $config->get('pat_on_merge');
+        if ('none' === $pat || ($configuredPat && 'none' === $configuredPat)) {
```

Comment:
No need to check if `$configuredPat` because even when it's null the second part will not match anyway.
Basically you only check if `$configuredPat is equal to 'none'` so anything else will never equal.